site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -23,11 +23,12 @@ layout: ../../layouts/ComponentLayout.astro
 
 Following [invokers](../invokers.explainer), adding an `interesttarget`
 attribute to interactive elements: starting with `<button>`, `<input
-type="button">`/`<input type="reset">`, and `<a>` (perhaps expanding to
-`<area>`, `<input>`, `<textarea>`, `<select>`, `<summary>` or maybe more, see
+type="button">`/`<input type="reset">`/`<input type="submit">`, `<a>`
+ and `<area>` (perhaps expanding to
```

Comment:
Area is included further down the page so this just fixes that here. Submit buttons are similarly mentioned as supported so make that explicit here.